client/my-sites/plans-features-main/index.tsx

```diff
@@ -754,6 +757,52 @@ const PlansFeaturesMain = ( {
 		</div>
 	);
 
+	const usePlanSlugsForAllDisplayedIntervals = () => {
```

Comment:
`Non-blocking / Nitpick`:

I'm seeing that we have two conventional ways of describing the length of a plan. Just to confirm my understanding, is the difference between "Displayed Interval" and "term" that one describes a UI concern while the other describes a business concern?

Reply:
I think one refers to the intervals in the toggle and the URL parameters, the other to the plan term/period.